pylsp/python_lsp.py

```diff
@@ -91,32 +91,116 @@ def start_io_lang_server(rfile, wfile, check_parent_process, handler_class):
     server.start()
 
 
+def start_ws_lang_server(bind_addr, port, check_parent_process, handler_class):
+    if not issubclass(handler_class, PythonLSPServer):
+        raise ValueError('Handler class must be an instance of PythonLSPServer')
+
+    # pylint: disable=import-outside-toplevel
+
+    # imports needed only for websocket based server
+    try:
+        from autobahn.twisted.websocket import WebSocketServerProtocol, WebSocketServerFactory
+        from twisted.internet import reactor
+        import ujson as json
```

Comment:
Please move the `ujson` import to the beginning of the file because it's a main dependency of this project, so you don't to check for its presence.